### ISSUE TITLE
.hack//G.U. Last Recode - 0100BA9014A02000

### ISSUE THREAD

**Issue by EliEron** [Ryujinx/Ryujinx-Games-List#3856]
#### Current on `master` : 1.1.65

Softlocks the moment it gets to the title screen. The music for the title screen plays, but the screen is completely frozen.

Spams a **TREMENDUS** amount of this warning:

```
HLE.GuestThread.15 KernelSvc PrintResult: WaitSynchronization64 returned error InvalidHandle.
```

I literally ended up with a 1.6GB log file after leaving the game running for just a minute or two. I did not know Ryujinx was capable of generating so much text so quickly, you learn something new everyday😄.

While I could upload a file like that as it compresses very well, I decided to instead manually trim most of the duplicate warnings to keep the file size to a sane level. I did not remove anything beyond the chain of duplicate messages so it shouldn't affect debugging in any way.

#### Screenshots :
![Ryujinx_hack_title](https://user-images.githubusercontent.com/22575945/157849884-20bc8739-a147-436c-8e99-8dd9710344b6.png)

#### Log file :
[HGULR.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/8231124/HGULR.log)

**Comment by MetrosexualGarbodor** [comment]
According to your log you have resolution set to 8x. Does it work on native?

**Comment by EliEron** [comment]
I was running the game at native res:

```00:00:00.088 |I| Configuration LogValueChange: ResScale set to: 1```

My custom res is set to 8x, but that only applies if I actually set the resolution to custom. Which would show up as: ```ResScale set to: -1``` in the log.

**Comment by MetrosexualGarbodor** [comment]
My mistake.

**Comment by Cheezballjones** [comment]
Since merge as of minutes ago - The game is operating perfectly. I have played for over an hour on 2x Res Scale and nothing has been out of the ordinary. I've only noticed that the videos that play seem a bit pixelated at times but I would assume that's just because its an old video from PS2 Era!

Just my 2 cents

**Comment by EliEron** [comment]
Indeed, I have updated my report to reflect the current compatibility. I can't change the labels but I'm sure MetrosexualGarbodor or one of the other maintainers will do so soon.

Also yes, the video blockiness is definitively due to the PS2 era files, they even include a warning about that right after booting the game:
![Ryujinx_ICblgb9fyC](https://user-images.githubusercontent.com/22575945/158031721-f7dd3b06-3084-43de-a655-0956906fd4db.png)

Which I found pretty humorous, but given just how blocky those movies are I guess the warning makes sense.
